app/models/criterium.rb

```diff
@@ -4,7 +4,36 @@ class Criterium < ApplicationRecord
 
   validates_inclusion_of :boundary, :five_year_commitment, :progress_report, in: [true, false], message: :nil_value
 
+  validates :cbd_objectives, presence: true
+  validates :stakeholders, presence: true
+  validate :none_of_the_above_for_cbd_commitments
+  validate :none_of_the_above_for_stakeholders
+
   def criteria_valid_for_commitment?
-    ![boundary, five_year_commitment, progress_report, cbd_objective_ids, stakeholder_ids].map(&:blank?).include?(true)
+    !criteria_invalid_for_commitment?
+  end
+
+  def criteria_invalid_for_commitment?
+    [boundary, five_year_commitment, progress_report, cbd_objective_ids, stakeholder_ids].map(&:blank?).include?(true) ||
+      stakeholders.pluck(:name).include?('Sub-national or local government') ||
+        cbd_objectives.pluck(:name).include?('None of the above')
```

Comment:
Is there a reason this is being checked here and in `none_of_the_above_for_cbd_commitments`, whereas for stakeholders it's just in `none_of_the_above_for_stakeholders`?